assets/sass/_cards.scss

```diff
@@ -72,6 +72,15 @@
 .card-content {
   font-weight: 300;
   text-transform: none;
+
+  h4 {
+    hyphens: auto;
+  }
+
+  p {
+    line-height: 20px;
```

Comment:
I'm wondering if we can compromise on the line-height to 24px?  The original was too "spacey" and 20px is too "crampy". :)